### PR TITLE
Required scripts and symlink for cli

### PR DESCRIPTION
**Pull request by harshad16**
Required scripts and symlink for cli
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

**Review by fridex — APPROVED** [review]
Thanks! :+1:

File: app.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+#
+# This script is run by OpenShift's s2i. Here we guarantee that we run desired
+# sub-command based on env-variables configuration.
+#
+
+case $THOTH_BUILD_ANALYSER_SUBCOMMAND in
+	'analyse')
+		exec /opt/app-root/bin/python3 thoth-build-analyser analyse
```
Comment (CermakM) [inline app.sh:9]: Just a suggestion: we might want to change the name of the script in `setup.py` from `thoth-build-analysis` to `thoth-build-analyser` to be uniform in the naming as we're using that all over the place.

**Review by CermakM — APPROVED** [review]

File: app.sh (hunk 1 above)
Comment (harshad16) [reply app.sh:9]: That's a really good suggestion, having consistency in the module is correct in my opinion.
i was thinking we should have consistency in the modules as well, i.e we have analyzer(in American English) and build-analysers(in British English).
It makes sense to me to have `build-analysers` **->** `build-analyzer`

What do you think @fridex  @CermakM ?

Comment (fridex) [reply app.sh:9]: +1 on my side.

Comment (CermakM) [reply app.sh:9]: +1

**Comment by fridex** [comment]
@harshad16 do you want to change name in this branch or create a separate PR and have this PR merged?

**Comment by harshad16** [comment]
> @harshad16 do you want to change name in this branch or create a separate PR and have this PR merged?

I was thinking to raise another pr, once these are merged. :smile: @fridex

**Pull request merged**